src/catalog/src/durable/impls/persist.rs

```diff
@@ -199,6 +201,9 @@ impl UnopenedPersistCatalogState {
         // Only writable catalogs attempt to increment the epoch.
         if matches!(mode, Mode::Writable) {
             current_epoch = current_epoch + 1;
+            if let Some(epoch_lower_bound) = epoch_lower_bound {
```

Comment:
might be worth putting an info log here? should be pretty high signal low noise. ditto in the other impl

Reply:
Done